src/JavascriptTrait.php

```
@@ -0,0 +1,321 @@
+   * Assert that no JavaScript errors were collected.
+   *
+   * @throws \Exception
+   *   If JavaScript errors were detected.
+   */
+  protected function javascriptAssertNoErrors(): void {
+    if (empty($this->javascriptErrorRegistry)) {
+      return;
+    }
+
+    // Build detailed error message.
+    $error_count = 0;
```

PHPMD (2.15.0)

The variable $error_count is not named in camelCase. (undefined)

(CamelCaseVariableName)